tox.ini

```diff
@@ -24,6 +25,7 @@ changedir =
   test-ext-wsgi: ext/opentelemetry-ext-wsgi/tests
 
 commands_pre =
+  pip install -U pip setuptools
```

Comment:
Just to be safe, I would add `wheel` there too.

Reply:
It's done.